content/en/news/security/istio-security-2021-001/index.md

```diff
@@ -13,12 +13,13 @@ skip_seealso: true
 
 {{< security_bulletin >}}
 
-This issue only affects Istio 1.9.0; previous versions of Istio are not affected. This issue has been given a CVSS score
-of 8.2 by the Istio product security working group.
+This issue only affects Istio 1.9.0; previous versions of Istio are not affected.
```

Comment:
Is this redundant? We used to have this, but I don't recall it in the recent bulletins. The last row in the table already says this.

Reply:
Yeah, we don't need this. Removed